samples/Basic_ProgMem/app/TestProgmem.cpp

```diff
@@ -123,7 +123,7 @@ void testFSTR(Print& out)
 	}
 	out.println("}");
 
-	// Test equality operators
+// Test equality operators
```

Comment:
Don't modify this file in the current PR.

Reply:
Fixed